tests/integration_tests/build/test_unittests.py

```diff
@@ -12,7 +13,7 @@
 # run coverage with the `gnu` toolchains and run unit tests with the `musl` toolchains.
 TARGET = "{}-unknown-linux-musl".format(MACHINE)
 
-
+@pytest.mark.timeout(600)
```

Comment:
the commit message says update code coverage so, this change seems to be in the wrong commit.

Reply:
I've mode it to a separate commit c76bba12adf68e5c2e811c33744c4cbc5494ef2d.